proof-libs/fstar/core/Core.Core_arch.fsti

```diff
@@ -0,0 +1,3 @@
+module Core.Core_arch
+
+include Core.Core_arch.X86
```

Comment:
I don't think we need this `include` here: generated Rust will never have `Core.Core_arch.t___m256i`, always `Core.Core_arch.X86.t___m256i`.
```suggestion
```